### PR TITLE
[servus] fix for new url format

### PR DESCRIPTION
**Pull request by Oneboy1979**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [X] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [X] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

File: youtube_dl/extractor/servus.py [hunk 1]
```diff
@@ -5,26 +5,24 @@
...
 class ServusIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?servus\.com/(?:at|de)/p/[^/]+/(?P<id>AA-\w+|\d+-\d+)'
+    _VALID_URL = r'https?://(?:www\.)?servus\.com/tv/videos/(?P<id>aa-\w+|\d+-\d+)'
     _TESTS = [{
-        'url': 'https://www.servus.com/de/p/Die-Gr%C3%BCnen-aus-Sicht-des-Volkes/AA-1T6VBU5PW1W12/',
+        'url': 'https://www.servus.com/tv/videos/aa-1t6vbu5pw1w12/',
         'md5': '046dee641cda1c4cabe13baef3be2c1c',
         'info_dict': {
-            'id': 'AA-1T6VBU5PW1W12',
+            'id': 'aa-1t6vbu5pw1w12',
```
Comment (dstftw) [inline youtube_dl/extractor/servus.py]: Ids must stay exactly the same.

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: The ID is part of the link, and is the same

Comment (dstftw) [reply youtube_dl/extractor/servus.py]: `AA-1T6VBU5PW1W12` != `aa-1t6vbu5pw1w12`.

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: Ok i fixed the test, sorry for the question, now it is clear!

File: youtube_dl/extractor/servus.py [hunk 2]
```diff
@@ -5,26 +5,24 @@
...
+    _TEST = {
+        'url': 'https://www.servus.com/tv/videos/aa-1t6vbu5pw1w12/',
+        'md5': '3e1dd16775aa8d5cbef23628cfffc1f4',
         'info_dict': {
             'id': 'AA-1T6VBU5PW1W12',
             'ext': 'mp4',
-            'title': 'Die Grünen aus Volkssicht',
-            'description': 'md5:052b5da1cb2cd7d562ef1f19be5a5cba',
-            'thumbnail': r're:^https?://.*\.jpg$',
+            'title': 'Die Grünen aus Sicht des Volkes - Servus TV',
```
Comment (dstftw) [inline youtube_dl/extractor/servus.py]: ` - Servus TV` should not be in title.

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: But it is in the title:

AssertionError: Invalid value for field title, expected u'Die Gr\xfcnen aus Sicht des Volkes', got u'Die Gr\xfcnen aus Sicht des Volkes - Servus TV'

Comment (dstftw) [reply youtube_dl/extractor/servus.py]: Again: it's not part of the title and must not be in the title.

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: Again: It is part of the title

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: See here: <meta property="og:title" content="Die Grünen aus Sicht des Volkes - Servus TV" />

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: `<meta property="og:title" content="Die Grünen aus Sicht des Volkes - Servus TV" />`

Comment (dstftw) [reply youtube_dl/extractor/servus.py]: I'm not going to argue. You either implement it as requested or it's not going to be merged.
image

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: Look at the source of the Website, the title has servus tv in there

Comment (Oneboy1979) [reply youtube_dl/extractor/servus.py]: I will change title to regex, but i cannot push this

**Pull request closed without merging**